cs/src/Contracts/TunnelAccessControlEntry.cs

```diff
@@ -127,12 +136,15 @@ public TunnelAccessControlEntry()
         /// with any other types of ACEs.
         /// </remarks>
         [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+        [StringLength(OrganizationMaxLength)]
```

Comment:
Are there valid or invalid chars defined for this property,? E.g. `<,` `&`, `/`, `\`, `"`, `>` are most likely invalid. 
Limiting these string contents and prohibiting invalid input will help mitigate stored or reflected XSS ([Types of XSS](https://owasp.org/www-community/Types_of_Cross-Site_Scripting)]

Same for other properties that don't have a regular expression validation.
Not blocking for this PR, but still a good thing to have, defense in depth.

Reply:
Yes, `Organization` and `Subject` here should use the same validation regex as in `TunnelAccessSubject`. I forgot to apply it here.

Reply:
Fixed.